app/assets/javascripts/cardboard/new_tab_links.js

```diff
@@ -0,0 +1,10 @@
+$(document).on("page:update", function() {
+  $("a").click(function() {
+    var link_host = this.href.split('/')[3];
+    var document_host = document.location.href.split('/')[3];
```

Comment:
I guess I was confused by the typical case in which this is done and the variable names -- could we change the variable names since it's not looking at a hostname at all? Or add a comment?